torch/csrc/jit/script/inline_loop_condition.cpp

```diff
@@ -0,0 +1,80 @@
+
+#include <functional>
+#include <memory>
+#include <string>
+
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/csrc/jit/ir.h>
+#include <torch/csrc/jit/script/inline_loop_condition.h>
+
+namespace torch {
+namespace jit {
+namespace script {
+
+namespace {
+
+void moveBlockBeforeNode(Node* before_node, Block* block) {
```

Comment:
Nitpick: the name of this function seemed a bit weird to me because blocks and nodes are never in the same list and thus it didn't make sense to move a block before a node. My suggestion is to use "inline" instead of "move".

Reply:
Good point